### PR TITLE
DAT-2553 Tag gallery edits and handle them in stat gathering script

### PR DESCRIPTION
**Pull request by sebastianmarzjan**

### CODE REVIEW

**Comment by idradm** [comment]
Could we fixed this mixed notation? Lets use camel case instead of underscores.

File: extensions/wikia/EditTagging/EditTaggingHooks.class.php [hunk 1]
```diff
@@ -69,12 +73,18 @@ public static function onArticleSaveComplete( &$article, &$user, $text, $summary
...
-		self::tagRevisionIfRTESourceEdit( $revision_id, $rte_mode );
-		self::tagRevisionIfRTEWysiwygEdit( $revision_id, $rte_mode );
+		// Iterate over all handlers until out of handlers or a handler returns true
+		// for this request
+		$handler_iterator = self::getHandlers()->getIterator();
+		while ( $handler_iterator->valid() ) {
+			$result = call_user_func( $handler_iterator->current(), $revision_id, $request );
+			if ( $result ) {
+				break;
+			} else {
```
Comment (idradm) [inline extensions/wikia/EditTagging/EditTaggingHooks.class.php]: Isn't this else unnecessary?

**Comment by SebastianMarzjan** [comment]
Yes, we could :)

File: extensions/wikia/EditTagging/EditTaggingHooks.class.php (hunk 1 above)
Comment (SebastianMarzjan) [inline extensions/wikia/EditTagging/EditTaggingHooks.class.php]: Yes. I had it here for readability, but it's clear what happens without it so I removed it. Thanks!

**Comment by idradm** [comment]
:shipit:

**Pull request merged**